source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRPresentationViewController.m

```diff
@@ -0,0 +1,44 @@
+//
+//  ACRPresentationViewController.m
+//  ADCIOS
+//
+//  Created by jwoo on 5/31/17.
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "ACRPresentationViewController.h"
+#import "ACRViewController.h"
+
+@implementation ACRPresentationViewController
+
+- (id) init:(NSString*) str{
+    self = [super init];
+    
+    if(self){
+        _jsonStr = nil;
+        _jsonStr = str;
+    }
+    
+    return self;
+}
+
+- (void) viewDidLoad {
+
+    [super viewDidLoad];
+    
+    if(self.jsonStr){
+        ACRViewController* acVc = [[ACRViewController alloc] init: self.jsonStr];
```

Comment:
Although I'm actually curious, what's the purpose of this view controller? I don't see what functionality it adds to the underlying ACRViewController..

---
In reply to: [122548933](https://github.com/Microsoft/AdaptiveCards/pull/423#discussion_r122548933) [](ancestors = 122548933)